manga_translator/args.py

```diff
@@ -88,11 +88,12 @@ def _format_action_invocation(self, action: argparse.Action) -> str:
 parser.add_argument('--inpainter', default='lama_mpe', type=str, choices=INPAINTERS, help='Inpainting model to use')
 parser.add_argument('--upscaler', default='esrgan', type=str, choices=UPSCALERS, help='Upscaler to use. --upscale-ratio has to be set for it to take effect')
 parser.add_argument('--upscale-ratio', default=None, type=int, choices=[1, 2, 4, 8, 16, 32], help='Image upscale ratio applied before detection. Can improve text detection.')
+parser.add_argument('--skip', default=None, type=str, choices=["detection", "ocr", "inpaint", "translation"], help='Stops after selected section')
```

Comment:
Ah it probably didnt make sense to add it after all because the only reason why somebody would use it is for getting an inpainted image. And that you can get with --translator none. Sorry can you remove it again?